setup.py

```diff
@@ -35,7 +35,43 @@
     "ipykernel>=6.29.0",
 ]
 
-rag = ["sentence_transformers", "pypdf", "ipython", "beautifulsoup4", "markdownify"]
+retrieve_chat = ["chromadb", "sentence_transformers", "pypdf", "ipython", "beautifulsoup4", "markdownify"]
+
+extra_require = {
+    "test": [
+        "coverage>=5.3",
+        "ipykernel",
+        "nbconvert",
+        "nbformat",
+        "pre-commit",
+        "pytest-asyncio",
+        "pytest>=6.1.1,<8",
+        "pandas",
+    ],
+    "blendsearch": ["flaml[blendsearch]"],
+    "mathchat": ["sympy", "pydantic==1.10.9", "wolframalpha"],
+    "retrievechat": retrieve_chat,
+    "retrievechat-pgvector": [
```

Comment:
@thinkall  this change was made because there is a hard dependency on chromadb through retrievechat_user_proxy and retrieve-utils. Until that dependency is removed, we also need to install chromadb with qdrant and pgvector.

Reply:
OK. I'd prefer remove chromadb from the `retrieve_chat` list and add it to each of the extras, as ideally, a db should not depend on another db.